src/main/java/org/jabref/gui/preferences/AdvancedTab.java

```diff
@@ -54,26 +54,12 @@ public AdvancedTab(DialogService dialogService, JabRefPreferences prefs) {
         Label remoteOperation = new Label(Localization.lang("Remote operation"));
         remoteOperation.getStyleClass().add("sectionHeader");
         builder.add(remoteOperation, 2, 1);
-        builder.add(new Separator(), 2, 1);
-        builder.add(new Pane(), 1, 2);
-        Label label1 = new Label(Localization.lang("This feature lets new files be opened or imported into an "
-
-                + "already running instance of JabRef<BR>instead of opening a new instance. For instance, this "
-
-                + "is useful when you open a file in JabRef<br>from your web browser."
-
-                + "<BR>Note that this will prevent you from running more than one instance of JabRef at a time."));
-        label1.setVisible(false);
-        builder.add(label1, 2, 22);
-
-        Label textLabel1 = new Label("    This feature lets new files be opened or imported into an already running instance of JabRef instead of opening a new instance. For");
-        builder.add(textLabel1, 2, 3);
-        Label textLabel2 = new Label("instance, this is useful when you open a file in JabRef from your web browser. ");
-        builder.add(textLabel2, 2, 4);
-        Label textLabel3 = new Label("    Note that this will prevent you from running more than one instance of JabRef at a time.");
-        builder.add(textLabel3, 2, 5);
-        builder.add(new Line(), 2, 6);
-        builder.add(new Pane(), 2, 7);
+        Text textRemote = new Text(
```

Comment:
Please add a `Localization.lang` call around the text so that it gets translated properly if the user changes the language. This should also take care of the failing travis build.

Reply:
Thanks, I changed it but it doesn't work. I've some issues when I just change little things, most frequently this one 

![Capture du 2019-04-25 21-27-32](https://user-images.githubusercontent.com/49909555/56762963-6dbcdc00-67a1-11e9-97af-2e97e4121dfb.png)
:

Reply:
This should not be related to your changes. Have you merged in the latest Master? 
And regarding the l10n, just run the Localisation Consistency Test to check if you need to remove or add new translation items